server/index-node/src/schema.graphql

```diff
@@ -18,6 +18,7 @@ type Query {
     blockNumber: Int!
     blockHash: Bytes!
     indexer: Bytes
+    accessToken: String
```

Comment:
I'd prefer the access token to be provided via a `Authorization: Bearer <token>` header because adding it here is a breaking change to the `proofOfIndexing` resolver. `Bearer` may not strictly be needed but IIRC clients often assume bearer tokens are used by default.

Reply:
Makes sense, I should be able to get this sorted by tomorrow.